app/views/audit.scala.html

```diff
@@ -782,10 +792,37 @@ <h1 class="text-center bold">@Messages("surface.problem")</h1>
         <div class="row">
             <div class="col-md-12">
                 <p>@Html(Messages("audit.tutorial.end.1"))</p>
-                <p>
-                    <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02.png")' id="onboarding-example-image-2"
-                    alt="Examples of accessibility attributes: curb ramps, missing curb ramps, obstacles in path, and surface problems.">
-                </p>
+                
+                
+                <table class="tutorial-table">
+                    <tr>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.accessible"))</th>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.inaccessible"))</th>
+                    </tr>
+                    <tr>
+                        <td colspan=2><img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02_top.png")' id="onboarding-example-image-2"
+                            alt="@Messages("audit.tutorial.examples")"></td>
+                    </tr>
+                    <tr>
+                        <td class="tutorial-table-cell">@Html(Messages("audit.tutorial.examples.curb.ramp"))</td>
+                        <td class="tutorial-table-cell">@Html(Messages("audit.tutorial.examples.missing.ramp"))</td>
+                    </tr>
+                    <tr height="15px"></tr>
+                    <tr>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.inaccessible"))</th>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.inaccessible"))</th>
+                    </tr>
+                    <tr>
+                        <td colspan=2><img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02_bot.png")' id="onboarding-example-image-2"
+                            alt="@Messages("audit.tutorial.examples")" ></td>
+                    </tr>
+                    <tr>
+                        <td class="tutorial-table-cell">@Html(Messages("audit.tutorial.examples.obstacle"))</td>
+                        <td class="tutorial-table-cell">@Html(Messages("audit.tutorial.examples.surface.problem"))</td>
+                    </tr>
+                </table>
+                
+                <p></p>
```

Comment:
What's with the empty `<p>` tags here? If you are trying to add space between elements, you should try to do that with CSS. Same with the `<tr height="15px"></tr>` above. I doubt that you actually want an empty row, you probably just want to add a margin/padding between rows.

Reply:
You can't add padding or margin to table rows unfortunately. I found an attribute called 'border-spacing' which I will try.

Reply:
Oh wait maybe its only margin that you can't use

Reply:
Okay I just added the padding to each element in the row, that works.